internal/cli/new.go

```diff
@@ -19,21 +20,36 @@ func newNewCmd(workdir string) *cli.Command {
 				if err := os.Mkdir(pathArg, 0755); err != nil {
 					return err
 				}
-				return createNevaMod(pathArg)
+				return createNevaMod(pathArg, "hello-world")
 			}
-			return createNevaMod(workdir)
+			return createNevaMod(workdir, "hello-world")
+		},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "template",
+				Usage:    "Specify the template for creating new projects.",
+				Required: false,
+
+				Action: func(context *cli.Context, templateName string) error {
+					if templateName == "" {
+						templateName = "hello-world"
+					}
+					pathArg := context.Args().First()
+					if err := os.Mkdir(pathArg, 0755); err != nil {
+						return err
+					}
+					return createNevaMod(pathArg, templateName)
```

Comment:
Did you test it? Asking because I was thinking `Action` work different. Looks like we call `createNevaMod` two times: from action of the command and form action of the flag. Take a look at how I use flag-actions in other commands (just to set variable and then use it command-action)